Fixes Issue #15

HaoZhang615 opened the pull request:
enhance deployment scripts by adding powershell scripts in preprovision, predown, postdeploy and postprovision for Azure CLI integration across platform (Windows, Linux and MacOS) and updated README.md.

HaoZhang615 commented:
also fixed another issue I encountered by making sure the requirements.txt in scripts/ folder gets installed before the db scripts are run.
<img width="329" alt="image" src="https://github.com/user-attachments/assets/5adf3971-a59d-4c2a-8e1f-8fee9922d52a" />

asevillano commented:
Hi,

I deleted the moneta-agents directory and clone the repo again, but the
command 'azd up' is still failing with the same error that I reported in
bug #15:
[image: image.png]

Any idea to fix it?

Thanks!

El vie, 7 feb 2025 a las 20:22, Hao Zhang ***@***.***>)
escribió:

> also fixed another issue I encountered by making sure the requirements.txt
> in scripts/ folder gets installed before the db scripts are run.
> image.png (view on web)
> <https://github.com/user-attachments/assets/5adf3971-a59d-4c2a-8e1f-8fee9922d52a>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Azure-Samples/moneta-agents/pull/27#issuecomment-2643830549>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEEINSJCKRO74JFLGLDQYUT2OUBY5AVCNFSM6AAAAABWWUFXZSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBTHAZTANJUHE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

HaoZhang615 commented:
Hi, Probably because the pull request has not yet been merged to main. Can you try to do the same from this fork of mine https://github.com/HaoZhang615/moneta-agents, would be a good validation as well, thanks!Cheers,HaoSent from my iPhoneOn 8 Feb 2025, at 20:52, Angel Sevillano ***@***.***> wrote:﻿
Hi,



I deleted the moneta-agents directory and clone the repo again, but the

command 'azd up' is still failing with the same error that I reported in

bug #15:

[image: image.png]



Any idea to fix it?



Thanks!



El vie, 7 feb 2025 a las 20:22, Hao Zhang ***@***.***>)

escribió:



> also fixed another issue I encountered by making sure the requirements.txt

> in scripts/ folder gets installed before the db scripts are run.

> image.png (view on web)

> <https://github.com/user-attachments/assets/5adf3971-a59d-4c2a-8e1f-8fee9922d52a>

>

> —

> Reply to this email directly, view it on GitHub

> <https://github.com/Azure-Samples/moneta-agents/pull/27#issuecomment-2643830549>,

> or unsubscribe

> <https://github.com/notifications/unsubscribe-auth/AEEINSJCKRO74JFLGLDQYUT2OUBY5AVCNFSM6AAAAABWWUFXZSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBTHAZTANJUHE>

> .

> You are receiving this because you are subscribed to this thread.Message

> ID: ***@***.***>

>



—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
